bats/fb-content-katello.bats

```diff
@@ -284,3 +284,191 @@ setup() {
   CONTAINER_PULL_LABEL=`echo "${ORGANIZATION_LABEL}-${PRODUCT_LABEL}-${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]'`
   podman pull "${HOSTNAME}/${CONTAINER_PULL_LABEL}"
 }
+
+@test "create and sync modules-rpms repo" {
+  hammer repository create --organization="${ORGANIZATION}" \
+    --product="${PRODUCT}" --content-type="yum" --name "${YUM_REPOSITORY_2}" \
+    --url https://partha.fedorapeople.org/test-repos/separated/modules-rpms/ | grep -q "Repository created"
+  hammer repository synchronize --organization="${ORGANIZATION}" \
+    --product="${PRODUCT}" --name="${YUM_REPOSITORY_2}"
+}
+
+@test "create and sync rpm-deps repo" {
+  hammer repository create --organization="${ORGANIZATION}" \
+    --product="${PRODUCT}" --content-type="yum" --name "${YUM_REPOSITORY_3}" \
+    --url https://partha.fedorapeople.org/test-repos/separated/rpm-deps/ | grep -q "Repository created"
+  hammer repository synchronize --organization="${ORGANIZATION}" \
+    --product="${PRODUCT}" --name="${YUM_REPOSITORY_3}"
+}
+
+@test "create first component content view" {
+  hammer content-view create --organization="${ORGANIZATION}" \
+    --name="${CONTENT_VIEW_2}" | grep -q "Content view created"
+}
+
+@test "add yum and docker repos to first component content view" {
+  repo_id=$(hammer repository list --organization="${ORGANIZATION}" \
+    | grep ${YUM_REPOSITORY} | cut -d\| -f1 | egrep -i '[0-9]+')
```

Comment:
Does hammer have an output mode that's easier to parse?

Reply:
you might look into  `hammer --csv --no-headers`

Reply:
I think you can have it spit it out as JSON.

Reply:
`hammer --csv --no-headers` looked like it made for the easiest parsing.  Just had to tokenize by `,` and use the right column